src/player.rs

```diff
@@ -91,6 +91,7 @@ fn player_shoot(
             .insert(Bullet {
                 speed: 0.2,
                 direction: target_dir,
+                origin_team: ChickenOrDog::Chicken,
```

Comment:
I would prefer to add chicken or dog as a component over encapsulating it in the bullet